src/application/tabs.css

```diff
@@ -99,18 +99,18 @@
 }
 
 
-.p-TabBar.p-mod-top .p-TabBar-tab {
+.p-TabBar.jp-mod-top .p-TabBar-tab {
   border-bottom: none;
 }
 
 
-.p-TabBar.p-mod-bottom .p-TabBar-tab {
+.p-TabBar.jp-mod-bottom .p-TabBar-tab {
   border-top: none;
 }
 
 
-.p-TabBar.p-mod-top .p-TabBar-tab:first-child,
-.p-TabBar.p-mod-bottom .p-TabBar-tab:first-child {
+.p-TabBar.jp-mod-top .p-TabBar-tab:first-child,
+.p-TabBar.jp-mod-bottom .p-TabBar-tab:first-child {
```

Comment:
All of these leading selectors with `p-mod-top` can be replaced by `.p-DockPanel-tabBar`, and then you dont need the custom renderer.